src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -53,7 +58,7 @@ public override async Task<ResourceElement> GetCapabilityStatementAsync(Cancella
                 {
                     if (_listedCapabilityStatement == null)
                     {
-                        ICapabilityStatementBuilder builder = CapabilityStatementBuilder.Create(_modelInfoProvider, _searchParameterDefinitionManager)
+                        ICapabilityStatementBuilder builder = CapabilityStatementBuilder.Create(_modelInfoProvider, _searchParameterDefinitionManager, _supportedProfiles)
```

Comment:
Since we cache the capability statement any new profiles posted will not show up unless we restart the server. Would it be useful to add a mechanism to regenerate the capability statement if a new profile is posted? For reference, we do something similar when reindex completes - we publish a `SearchParametersUpdated` notification and reset the capabilitystatement (down below) when we receive that notification.